src/wasm/utils.rs

```diff
@@ -178,3 +190,11 @@ pub fn log_benchmark(
         )
     );
 }
+
+pub fn node_read_file_sync(fs: &NodeFs, path: &str) -> Vec<u8> {
+    fs.readFileSync(path)
```

Comment:
why not use the async version? It's complex?

Reply:
Not complex, just have to create a new WASM endpoint, because at the moment I read the PNG and then blend. So, for 4 layers: read layer 1, read layer 2, blend, read layer 3, blend, ..., save to FS.

This is what is done in the single threaded pconvert pymodule `blend_multiple` version (the one currently used by the py light node). However, the `options` parameter accepts the number of threads which can make use of the multi threaded version. For now, to achieve the MVP I thought on post poning an async implementation of this WASM endpoint and later implement it similarly to the pymodule